src/hooks/useCommands.tsx

```diff
@@ -10,13 +9,18 @@ import {
 } from 'react'
 import { useParams } from 'react-router-dom'
 import { Command, System } from 'types/backend-types'
-import { AugmentedCommand, CommandIndexTableData } from 'types/custom-types'
+import {
+  AugmentedCommand,
+  CommandIndexTableData,
+  StrippedSystem,
+} from 'types/custom-types'
+import { generateCommandName } from 'utils/generateCommandName'
 
 export const useCommands = () => {
```

Comment:
This went through a refactor, but its API remained the same, so using code should not need to be updated.